infrastructure/tooling/src/build/tasks/livelog.js

```diff
@@ -72,9 +72,15 @@ module.exports = ({ tasks, cmdOptions, credentials, baseDir, logsDir }) => {
       // this simple Dockerfile just packages the binary into a Docker image
       const dockerfile = path.join(contextDir, 'Dockerfile');
       fs.writeFileSync(dockerfile, [
-        'FROM busybox', // We use busybox rather than scratch to have `/tmp` and such things
+        'FROM ubuntu:latest AS certs',
```

Comment:
no latest please :)

Reply:
I think this is an exceptional case - the fetch is for root certs only, and with the certificate chains, I think we _always_ want the latest version. Agreed that normally we would want to pin, but the risk of pinning certs here is that they will become out-of-date, and we'll only notice one day in production when suddenly certs are missing or invalid.

Reply:
ah, sure, we agreed that latest is fine for this :)